### ISSUE TITLE
Suffixes (and possibly prefixes)

### ISSUE THREAD

**Issue by ms-mialingvo** [asterics/AsTeRICS-Grid#404]
Option to mark a word as a suffix and if possible also as a prefix and add it accordingly to another word (so, without the space between the previous (suffix, e.g. sing+ing) respectively the following (prefix e.g. im+perfect) word).

**Comment by Fukurouonthesea** [comment]
Oh! Oh, this would be so good - you could, if I understand right, also use this for punctuation - like "+, +", +. and so on, so it adds them to the last word without a space.

**Comment by ms-mialingvo** [comment]
Yes, that would also work for proper punctuation without space in between.

**Comment by goldstargloww** [comment]
bumping this - this would be really useful :D
